cli/mender-artifact/modify_existing_test.go

```diff
@@ -294,7 +294,8 @@ func removeVolatileEntries(input string) string {
 	var output strings.Builder
 	for _, line := range strings.Split(input, "\n") {
 		if strings.HasPrefix(line, "      checksum:") ||
-			strings.HasPrefix(line, "      modified:") {
+			strings.HasPrefix(line, "      modified:") ||
+			strings.HasPrefix(line, "\trootfs_image_checksum:") {
```

Comment:
So is there actually like four tabs output in the preceding two cases, and a tab in the last one?

Reply:
Yes, surprising to me as well, but that's what's there! :-)